### DISCUSSION TITLE
Plugin seems not to be supported in OBS version 28.0

### DISCUSSION THREAD

**Discussion by Borvius** [cpyarger/obs-midi#201] [Q&A]
Hello. I recently updated my OBS installation to 28.0 and seems like that OBS version doesn't support the plugin. Is this something you guys are already aware of?

Thank you

**Comment by burningmonkapp** [comment]
I can't see OBS-MIDI in my Tools section either, but it's my first time installing – is this because it's not working in 28.0?

Thanks for your work!
